internal/remotestate/remote_state.go

```
@@ -130,7 +132,19 @@ func (remote *RemoteState) GetTFInitArgs() []string {
 	var backendConfigArgs = make([]string, 0, len(config))
 
 	for key, value := range config {
-		arg := fmt.Sprintf("-backend-config=%s=%v", key, value)
+		var sanitizedValue string
+		if reflect.ValueOf(value).Kind() == reflect.Map {
+			jsonValue, err := json.Marshal(value)
+			if err != nil {
+				// TODO: Log error
+				continue
+			}
+			sanitizedValue = string(jsonValue)
```

golangci-lint (1.64.8)

assignments should only be cuddled with other assignments

(wsl)